linux-support.md

```diff
@@ -0,0 +1,55 @@
+# .NET Support and Compatibility for Linux Distributions
+
+.NET is officially supported on multiple [Linux distributions](os-support-lifecycle.md) and [supported by the community on several others](linux.md).
+
+There are various aspects of compatibility to consider.
+
+## libc compatibility
+
+.NET supports both [glibc](https://www.gnu.org/software/libc/)- and [musl libc](https://musl.libc.org/)-based Linux distributions. [.NET release notes](release-notes/README.md) specify the minimum required libc version (for example for [.NET 6](release-notes/6.0/supported-os.md#linux)). You can use this version information to determine if a given .NET version will run on a given Linux distribution.
+
+On Alpine 3.16:
+
+```bash
+# ldd --version
+musl libc (aarch64)
+Version 1.2.3
+```
+
+On Ubuntu 22.04:
+
+```bash
+# ldd --version
+ldd (Ubuntu GLIBC 2.35-0ubuntu3.1) 2.35
+```
+
+## OpenSSL compatibility
+
+.NET 6 and later versions support both OpenSSL 1.x and 3.x and can be run on distributions with either version of this package. For example, Ubuntu 22.04 only includes OpenSSL 3 in its official package archive. Previous .NET versions only support OpenSSL 1.x.
+
+.NET has two policies for OpenSSL:
+
+- When built into a distro archive, .NET will load a specific OpenSSL version, per the distro archive policy.
+- When built by Microsoft (as a portable Linux build), .NET will the highest OpenSSL version it finds (that it supports), but can be configured to use a specific version.
```

Comment:
This is... true... but not always.

If we add support for a new OpenSSL via servicing it gets added as the lowest priority.  (IIRC, we added support for 1.1.x lower than 1.0.x because we did so late or in servicing; then made 1.1.x the higher priority in the next release)

Reply:
I didn't realize that. That's certainly good context, but it is also messy. Do you have a specific edit you'd like me to take?

Reply:
Not a specific edit, no.  Though throwing in a weasel word like "generally" is probably good enough.  

> When built by Microsoft (as a portable Linux build), .NET will **generally** use the highest OpenSSL version it finds (that it supports), but can be configured to use a specific version.

Perhaps?